pkg/api/types.go

```diff
@@ -313,6 +313,8 @@ const (
 type PersistentVolumeStatus struct {
 	// Phase indicates if a volume is available, bound to a claim, or released by a claim
 	Phase PersistentVolumePhase `json:"phase,omitempty"`
+	// Current service state of volume
+	Conditions []PersistentVolumeCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
```

Comment:
Also, there are docs in markdown with an idea of my understanding of conditions. Generally, it seemed that the 3 things that happen to a volume (recycle,delete,bind) were well expressed as conditions and those conditions could drive the lifecycle of that volume via controllers.